### PR TITLE
Add precache command

### PR DESCRIPTION
**Pull request by philippjfr**
Implements https://github.com/holoviz/lumen/issues/339

- [ ] Add tests
- [ ] Add docs

### CODE REVIEW

File: lumen/util.py [hunk 1]
```diff
@@ -15,9 +15,12 @@
 from jinja2 import DebugUndefined, Environment, Undefined
 from pandas.core.dtypes.dtypes import CategoricalDtype
 from panel import state
+from panel.io.cache import _hash_funcs
 
 log = getLogger(__name__)
 
+# Temporarily patch panel _hash_funcs
+_hash_funcs[dt.date] = lambda obj: f'{type(obj)}{obj}'.encode('utf-8')
```
Comment (Hoxbro) [inline lumen/util.py:23]: Why not just pin panel? Is it because `0.14.1` is needed?

Comment (philippjfr) [reply lumen/util.py:23]: Yes, will release 0.14.1 tomorrow or Monday so this will be removed before the Lumen release but we need this for a demo today.

Comment (philippjfr) [reply lumen/util.py:23]: Demo driven development is of course not ideal but don't have much choice in this case.

Comment (Hoxbro) [reply lumen/util.py:23]: I understand. 

I would properly open up a PR with the removal and a pin, so it is not forgotten before the next release. But maybe that is just me being too cautious.

**Pull request merged**